engine/gamesys/src/gamesys/components/comp_mesh.cpp

```diff
@@ -729,6 +733,46 @@ namespace dmGameSystem
         }
     }
 
+    static void RenderListFrustumCulling(dmRender::RenderListVisibilityParams const &params)
+    {
+        DM_PROFILE("Mesh"); // TODO - use a proper name here
+
+        MeshWorld* mesh_world = (MeshWorld*)params.m_UserData;
+
+        const dmIntersection::Frustum frustum = *params.m_Frustum;
+        uint32_t num_entries = params.m_NumEntries;
+        for (uint32_t i = 0; i < num_entries; ++i)
+        {
+            dmRender::RenderListEntry* entry = &params.m_Entries[i];
+            MeshComponent* component_p = (MeshComponent*)entry->m_UserData;
+
+            dmGameSystem::BufferResource* br = GetVerticesBuffer(component_p, component_p->m_Resource);
+            dmBuffer::Bounds3D* bounds = 0x0;
+            dmBuffer::Result r = dmBuffer::GetBounds(br->m_Buffer, &bounds);
+
+            if (r != dmBuffer::RESULT_OK) {
+                return;
+            }
```

Comment:
This will break the frustum culling for the rest of the items?

Reply:
yes, sloppy stuff. It has been fixed.